docs/content/preview/contribute/core-database/build-from-src-ubuntu.md

```diff
@@ -64,6 +64,9 @@ source <(cat /etc/os-release | grep '^VERSION_ID=')
 case "$VERSION_ID" in
   20.04)
     gcc_version=10
+    # 20.04 needs deadsnakes repo for python3.11 installed below
```

Comment:
You need to also explain in the `### Python 3` section below these instructions.